apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/(sidebar)/transactions/explorer/components/scalar.tsx

```diff
@@ -1,42 +0,0 @@
-"use client";
-import { ApiReferenceReact } from "@scalar/api-reference-react";
-import "@scalar/api-reference-react/style.css";
-import {
-  NEXT_PUBLIC_ENGINE_CLOUD_URL,
-  NEXT_PUBLIC_THIRDWEB_API_HOST,
-} from "@/constants/public-envs";
-
-interface ScalarProps {
-  useEngineAPI: boolean;
-}
-
-export function Scalar({ useEngineAPI }: ScalarProps) {
-  const apiUrl = useEngineAPI
-    ? NEXT_PUBLIC_ENGINE_CLOUD_URL
-    : NEXT_PUBLIC_THIRDWEB_API_HOST;
-  const openApiUrl = useEngineAPI
-    ? `${apiUrl}/openapi`
-    : `${apiUrl}/openapi.json`;
-  const apiName = useEngineAPI ? "Engine API (advanced)" : "thirdweb API";
-  const serverDescription = useEngineAPI ? "Engine Cloud" : "thirdweb API";
-
-  return (
-    <div className="flex flex-col gap-4">
-      <h2 className="font-bold text-2xl tracking-tight">
-        Full API Reference - {apiName}
-      </h2>
-      <ApiReferenceReact
```

Comment:
where did this go? or did we decided to nix it in favor of docs/link?

Reply:
nix it in favor of api reference & docs - there's no good spot for it here